includes/data-events/listeners.php

```diff
@@ -281,6 +279,54 @@ function ( $order_id, $order ) {
 	}
 );
 
+/**
+ * For non-donation purchases.
+ */
+Data_Events::register_listener(
+	'woocommerce_order_status_completed',
+	'product_order_new',
```

Comment:
suggestion: call the event `product_order_completed`

Reply:
I know we used `donation_new` but that actually created a problem, because when we wanted to differentiate new donations from renewal donation payments, the name was taken (donation_new fires for new donations and renewals).

Keeping it complete makes it more tied to what exactly this is and leaves room for other events we might want to add in the future

Reply:
Updated ([details](https://github.com/Automattic/newspack-plugin/pull/2821#issuecomment-1888116911))